{
  "Header": {
    "Original": "CVE-2020-28496 (This affects the package three before 0.125.0. This can happen when ha ...)",
    "Line": 188850,
    "ID": "CVE-2020-28496",
    "Description": "(This affects the package three before 0.125.0. This can happen when ha ...)"
  },
  "Annotations": [
    {
      "Original": "- three.js \u003cnot-affected\u003e (Vulnerable code introduced later, #988726)",
      "Line": 188851,
      "Type": "package",
      "Package": "three.js",
      "Kind": "not-affected",
      "Description": "Vulnerable code introduced later, #988726"
    },
    {
      "Original": "NOTE: https://github.com/mrdoob/three.js/pull/21143/commits/4a582355216b620176a291ff319d740e619d583e",
      "Line": 188852,
      "Type": "NOTE",
      "Description": "https://github.com/mrdoob/three.js/pull/21143/commits/4a582355216b620176a291ff319d740e619d583e"
    },
    {
      "Original": "NOTE: https://github.com/mrdoob/three.js/issues/21132",
      "Line": 188853,
      "Type": "NOTE",
      "Description": "https://github.com/mrdoob/three.js/issues/21132"
    }
  ]
}